db/migrate/20210910121223_create_uservotes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUservotes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :uservotes do |t|
+      t.integer :value
+      t.integer :votable_id
+      t.string :votable_type
```

Comment:
id belongs to a type. type should come before id

Reply:
add enum in model

Reply:
> id belongs to a type. type should come before id

changing the argument position in command changes the output file generated

earlier I was using this
`rails g model user_vote value:integer votable_id:integer votable_type:string user:references`

and if I change the order, type comes before id 
` rails d model user_vote value:integer votable_type:string votable_id:integer user:references`